backend/main.py

```diff
@@ -157,42 +158,42 @@ def get_new_info(search_term, is_initial=False):
     all_news_data = []
     # iterate pages to get more news data, not actually get all news data
     if is_initial:
-        a = []
-        for p in range(1, 10):
-            p2 = {
-                "page": p,
+        news_data = []
+        for page in range(1, 10):
+            page_metadata = {
+                "page": page,
                 "id": f"search:{quote(search_term)}",
                 "channelId": 2,
                 "type": "searchword",
             }
-            response = requests.get("https://udn.com/api/more", params=p2)
-            a.append(response.json()["lists"])
+            response = requests.get("https://udn.com/api/more", params=page_metadata)
+            news_data.append(response.json()["lists"])
 
-        for l in a:
-            all_news_data.append(l)
+        for news_list in news_data:
+            all_news_data.append(news_list)
```

Comment:
The variable should not be named with "list" if it's not a list type.

As I look, the all_news_data type is a type of list that contains another list.